frontend/src/Components/ProviderCard.tsx

```diff
@@ -53,64 +54,50 @@ export default function ProviderCard({
-                                    <div
-                                        className="tooltip"
-                                        data-tip="Manage Users"
-                                    >
-                                        <UserGroupIcon
-                                            className="w-4"
-                                            onClick={() =>
-                                                navigate(
-                                                    `/provider-users/${provider.id}`
-                                                )
-                                            }
-                                        />
-                                    </div>
                                 )}
                             </>
                         ) : (
-                            <div
-                                className="tooltip"
-                                data-tip="Register Provider"
-                            >
-                                <LinkIcon
-                                    className="w-4 "
-                                    onClick={() => oidcClient(provider)}
-                                />
-                            </div>
-                        )}
-                        <div className="tooltip" data-tip="Edit Provider">
-                            <PencilSquareIcon
-                                className="w-4"
-                                onClick={() => openEditProvider(provider)}
-                            />
-                        </div>
-                        <div
-                            className="tooltip"
-                            data-tip="Disable/Archive Provider"
-                        >
-                            <XMarkIcon
-                                className="w-4"
-                                onClick={() => archiveProvider(provider)}
+                            <ULIComponent
+                                dataTip={'Register Provider'}
+                                icon={LinkIcon}
+                                onClick={() => oidcClient(provider)}
                             />
-                        </div>
-                    </>
-                ) : (
-                    <div className="tooltip" data-tip="Enable Provider">
-                        <CheckCircleIcon
-                            className="w-4"
+                        )}
+                        <ULIComponent
+                            dataTip={'Edit Provider'}
+                            icon={PencilSquareIcon}
+                            onClick={() => openEditProvider(provider)}
+                        />
+                        <ULIComponent
+                            dataTip={'Disable/Archive Provider'}
+                            icon={XMarkIcon}
                             onClick={() => archiveProvider(provider)}
                         />
```

Comment:
sorry last thing- because this cuts off the tooltip, could we add `tooltipClassName="tooltip-left"
` onto here so you can fully read what it says? Once that is complete it should all be good to go & I'll approve and merge